src/main/java/uk/gov/ida/verifyserviceprovider/configuration/HubEnvironment.java

```diff
@@ -109,6 +111,14 @@ public String getEidasHubConnectorEntityId(){
         return this.eidasHubConnectorEntityId;
     }
 
+    public String[] getEidasAcceptableHubConnectorEntityIds() {
+        return ArrayUtils.isEmpty(eidasAcceptableHubConnectorEntityIds)
+            ? new String[] { eidasHubConnectorEntityId }
+            : ArrayUtils.contains(eidasAcceptableHubConnectorEntityIds, eidasHubConnectorEntityId)
+                ? eidasAcceptableHubConnectorEntityIds
+                : (String[])ArrayUtils.add(eidasAcceptableHubConnectorEntityIds, eidasHubConnectorEntityId);
+    }
```

Comment:
Would be better to do `unique set` or `unique list` or something like that. You can also use lambda nicely here along the lines of: create new list containing `hubConnectorEntityId `, then add `acceptableHubConnectorEntityIds ` and make a unique set 

Also we don't use `ArrayUtils` - this is a 3rd party library. Can we use `Arrays.asList` or `Arrrays.stream` instead so we stick to using Java built-ins?

Also it would help if `acceptableHubConnectorEntityIds` was a list - using an array is clunky